### PR TITLE
ppf_icp.py: Python 3 support

### PR DESCRIPTION
**Pull request by nszceta**

### CODE REVIEW

File: modules/surface_matching/samples/ppf_icp.py [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+cloud, rotated_cloud = [None] * 3, [None] * 3
+retval, residual, pose = [None] * 3, [None] * 3, [None] * 3
 noise = np.random.normal(0.0, 0.1, height * width * 3).reshape((-1, 3))
 noise2 = np.random.normal(0.0, 1.0, height * width)
 
 x, y = np.meshgrid(
-    range(-width/2, width/2),
-    range(-height/2, height/2),
+    range(-width // 2, width // 2),
+    range(-height // 2, height // 2),
```
Comment (alalek) [inline modules/surface_matching/samples/ppf_icp.py]: Please eliminate other unnecessary changes. No need to touch unrelated code.
Also proper description for this is: "ppf_icp.py: Python 3 support"

Comment (nszceta) [reply modules/surface_matching/samples/ppf_icp.py]: / to // is necessary. Python 3 range requires int not float

Comment (nszceta) [reply modules/surface_matching/samples/ppf_icp.py]: I am verifying why I needed to make this change to make the code work

Comment (nszceta) [reply modules/surface_matching/samples/ppf_icp.py]: Although not immediately apparent at a glance, type(-width/2) is float not int

Comment (alalek) [reply modules/surface_matching/samples/ppf_icp.py]: You need this change for Python3, but you don't need other 16 changes related to whitespace modifications.

**Comment by nszceta** [comment]
@alalek fixed. squashed. 

https://github.com/opencv/opencv_contrib/compare/master...nszceta:master
